envrc

```diff
@@ -13,7 +13,12 @@ export WBTC_ADDRESS="0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"
 export LON_ADDRESS="0x0000000000095413afC295d19EDeb1Ad7B71c952"
 export ANKRETH_ADDRESS="0xE95A203B1a91a908F9B9CE46459d101078c2c3cb"
 
+export ARBITRUM_L1_GATEWAY_ROUTER_ADDRESS="0x72Ce9c846789fdB6fC1f34aC4AD25Dd9ef7031ef"
+export ARBITRUM_L1_BRIDGE_ADDRESS="0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a"
+export OPTIMISM_L1_STANDARD_BRIDGE_ADDRESS="0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+
 export AMMWRAPPER_ADDRESS="0xCCaC49778C26bcC60e4648aB1E4C5F0276E82851"
 export AMMQUOTER_ADDRESS="0x77d8eeF8d15207b2228F76105cb46E558De7a8bD"
 export RFQ_ADDRESS="0x953CA2083aA94800d784ac37DEb3f15006bC6BD3"
 export LIMITORDER_ADDRESS="0x77d8eeF8d15207b2228F76105cb46E558De7a8bD"
+export L2DEPOSIT_ADDRESS="0x0B85B7d6bf392F665FBa4F4F5f42C645c7d9055B"
```

Comment:
Maybe add a comment to note where these contracts are deployed?

Reply:
You mean which environment? Actually this is an example file and should be adjusted before execute the deployment test.

Reply:
Got it. I misunderstood.